src/homework/server/services/ItemManager.js

```diff
@@ -0,0 +1,136 @@
+const pokemonClient = require("../clients/PokemonClient.js");
+const {Item, Sequelize} = require("../db/models")
+const fs = require("fs").promises;
+class ItemManager {
+  constructor() {
+    this.taskList = [];
```

Comment:
why do you need to keep all the db in your app memory? it's not scalable and not needed